packages/app/cypress.config.ts

```diff
@@ -2,11 +2,8 @@ import { defineConfig } from 'cypress'
 import getenv from 'getenv'
 import { initGitRepoForTestProject, resetGitRepoForTestProject } from './cypress/tasks/git'
 
-const CYPRESS_INTERNAL_CLOUD_ENV = getenv('CYPRESS_INTERNAL_CLOUD_ENV', process.env.CYPRESS_INTERNAL_ENV || 'development')
-const CYPRESS_INTERNAL_DEV_PROJECT_ID = getenv('CYPRESS_INTERNAL_DEV_PROJECT_ID', process.env.CYPRESS_INTERNAL_DEV_PROJECT_ID || 'sehy69')
-
 export default defineConfig({
-  projectId: CYPRESS_INTERNAL_CLOUD_ENV === 'staging' ? 'ypt4pf' : CYPRESS_INTERNAL_DEV_PROJECT_ID,
+  projectId: getenv('CYPRESS_INTERNAL_DEV_PROJECT_ID', process.env.CYPRESS_INTERNAL_DEV_PROJECT_ID || 'ypt4pf'),
```

Comment:
Do we still need the `getenv`? Why do most of our packages just have `projectId: 'ypt4pf'` but our frontend packages use all of the env variables.

Reply:
Not sure why... I left these in because I figure we should be able to override the project IDs with an environment variable if we need to, but yeah it looks like only these packages use this pattern

Reply:
@ZachJW34 I removed `getenv` and just hard-coded the values for projectId like we do in the other packages